### PR TITLE
Check if credentials already passed as argument

### PR DESCRIPTION
**Pull request by andreypolyak**
For those who don't like to store credentials in file system

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -78,11 +78,16 @@ Waker.prototype._doWake = function(device, callback) {
 
 Waker.prototype.readCredentials = function(callback) {
     
-    fs.readFile(this.credentials, function(err, buf) {
-        if (err) return callback(err);
-        
-        callback(null, JSON.parse(buf.toString()));
-    });
+    if (this.credentials !== null && typeof this.credentials === 'object') {
```
Comment (dhleong) [inline index.js]: Please use parens with `typeof()` so the style is consistent with the rest of the project

File: index.js [hunk 2]
```diff
@@ -78,11 +78,16 @@ Waker.prototype._doWake = function(device, callback) {
...
+        callback(null, this.credentials);
+    }
+    else {
```
Comment (dhleong) [inline index.js]: Similarly, the else should go on the same line as the previous closing bracket for consistency, eg:

```javascript
  } else {
```

**Comment by dhleong** [comment]
Cool stuff, thanks! If you could just revise the PR with the style changes I think this looks good.

**Comment by dhleong** [comment]
Sorry, didn't see the commit until just now. Thanks!

**Comment by dhleong** [comment]
I went ahead and published this a 0.7.6 so you can use it as a dependency if you like

**Comment by andreypolyak** [comment]
Great! Thank you!

**Pull request merged**